src/cli.ts

```diff
@@ -155,7 +157,16 @@ async function main(args: typeof cliFlags, config: Config) {
 					new Listr([getListrTask(file)], { exitOnError: false }).run().catch(console.error),
 				);
 			} else {
-				server.close();
+				await closeBrowser();
+				await closeServer(server);
+				/**
+				 * @todo why are there still open connections to the server even after closing the browser :/
+				 *
+				 * Can be inspected with `console.log((process as any)._getActiveHandles());`
+				 *
+				 * Maybe https://github.com/isaacs/server-destroy would fix it.
+				 */
+				process.exit();
```

Comment:
@chamabreu did you also encounter this? When converting only a single file, or when disabling concurrency, it would exit just fine, but when i converted 5 files concurrently, the process would hang forever, and `process._getActiveHandles()` would show some open (websocket) connections from the browser to the server.

Maybe it's a puppeteer bug, because after calling `browser.close()`, no connections should be left open. 🤷🏻‍♂️

Reply:
I will test this tomorrow.
I didn't experienced it yet.
Maybe close the server before the browser?

I didn't also inspected yet what the server is doing, I will do that. Maybe I find something.

Reply:
I ran it with 90 files, concurrently with cli.
no problem with hanging.
Ran on Arch Linux 6.0.6